methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        os.system(command);
</pre><p>This we keep as ARIA-tools cannot de-duplicate easy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I thought the intention would be to avoid rerunning tssetup if we already have the staged mintpy data? Therefore, we should skip this if the data is staged, no?</p>